src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC } from 'react';
+import cn from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { FilterBy } from '../../types/FilterBy';
+
+import { filterTodos } from '../../utils/FilterTodos';
+
+interface Props {
+  selectedFilter: FilterBy;
+  todos: Todo[];
+  onSelectFilter: (filter: FilterBy) => void;
+}
+
+const Footer: FC<Props> = ({ selectedFilter, todos, onSelectFilter }) => {
+  const activeTodos = filterTodos(todos, FilterBy.ACTIVE);
+  const completedTodos = filterTodos(todos, FilterBy.COMPLETED);
+
+  const filters = Object.values(FilterBy).filter(filter => filter.length);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodos.length} items left
+      </span>
+
+      {/* Active link should have the 'selected' class */}
```

Comment:
Remove all comments

Reply:
Refactor code according to your comments)